anndata/_io/specs/registry.py

```diff
@@ -326,6 +383,20 @@ def read_elem(elem: StorageType) -> Any:
     return Reader(_REGISTRY).read_elem(elem)
 
 
+def read_groups(
+    elem: StorageType,
+    modifiers: frozenset(str) = frozenset(),
+) -> Any:
+    """
+    Read an element from as Zarr group
+    """
+
+    reader = GroupReader(_REGISTRY)
+    return _REGISTRY.get_read_groups(type(elem), get_spec(elem), frozenset(modifiers))(
+        elem, _reader=reader
+    )
```

Comment:
I see what you mentioned in the meeting now. 

For now, I think I would accomplish this with a `read_dispatched` function.

But, if I were to do this through the registries, instead of creating a new method on the registry class, I would create a new registry.

Reply:
fixed that here https://github.com/scverse/anndata/pull/967